testo.d/garmin_fit.test

```diff
@@ -13,3 +13,6 @@ compare ${REFERENCE}/track/garmin-edge-800-output.gpx ${TMPDIR}/fit-sample-800.g
 
 gpsbabel -i garmin_fit -f ${REFERENCE}/track/garmin-forerunner-10.fit -o gpx -F ${TMPDIR}/fit-sample-10.gpx
 compare ${REFERENCE}/track/garmin-forerunner-10-output.gpx ${TMPDIR}/fit-sample-10.gpx
+
+gpsbabel -i garmin_fit,allpoints -f ${REFERENCE}/track/garmin-vivosmart-hr.fit -o gpx,garminextensions -F ${TMPDIR}/fit-sample-vivosmart-hr.gpx
```

Comment:
Ah, ok, that's why. I guessed that there was some reason for the points to be excluded, so I did my change as an option to not change the behavior in the default case.

I have added a short description of the option in xmldoc/formats/options/garmin_fit-allpoints.xml.